include/reactor-cpp/action.hh

```diff
@@ -15,159 +15,175 @@
 namespace reactor {
 
 class BaseAction : public ReactorElement {
- private:
-  std::set<Reaction*> _triggers;
-  std::set<Reaction*> _schedulers;
+private:
+    std::set<Reaction*> triggers_;
+    std::set<Reaction*> schedulers_;
 
-  const bool _logical;
+    const bool logical_;
 
- protected:
-  void register_trigger(Reaction* reaction);
-  void register_scheduler(Reaction* reaction);
+protected:
+    void register_trigger(Reaction* reaction);
+    void register_scheduler(Reaction* reaction);
 
-  virtual void cleanup() = 0;
+    virtual void cleanup() = 0;
 
-  const Duration min_delay;
+    const Duration min_delay;
 
- protected:
-  BaseAction(const std::string& name,
+    BaseAction(const std::string& name,
              Reactor* container,
              bool logical,
              Duration min_delay)
-      : ReactorElement(name, ReactorElement::Type::Action, container)
-      , _logical(logical)
-      , min_delay(min_delay) {}
+        : ReactorElement(name, ReactorElement::Type::Action, container)
+        , logical_(logical)
+        , min_delay(min_delay) {}
 
- public:
-  const auto& triggers() const { return _triggers; }
-  const auto& schedulers() const { return _schedulers; }
+public:
+    [[nodiscard]] auto triggers() const noexcept -> const auto& {
```

Comment:
Why does this change to the `->` notation? Is this recommended by the guidelines?

Reply:
no I don't think so but it's included inside the clang-tidy modernization check.